compiler/src/main/kotlin/io/github/apl_cornell/viaduct/selection/SelectionConstraintGenerator.kt

```diff
@@ -55,7 +55,7 @@ class SelectionConstraintGenerator(
     private val costEstimator: CostEstimator<IntegerCost>,
 ) {
     private val nameGenerator = FreshNameGenerator()
-    private val hostTrustConfiguration = HostTrustConfiguration(program)
+    private val hostTrustConfiguration = HostTrustConfiguration.get(program)
```

Comment:
What is the canonical way to get a `HostTrustConfiguration`? Using `.get` or `informationFlowAnalysis.trustConfiguration`?

Reply:
using .get(). I implemented the class the same way we did for analysis classes

Reply:
We should make `informationFlowAnalysis.trustConfiguration` private then.